.github/workflows/launcher.yml

```
@@ -15,7 +15,7 @@ jobs:
             artifact-name: Linux
             extension: deb
           - os: macos-latest
-            artifact-name: macOS
+            artifact-name: MacOS
```

```suggestion
            artifact-name: macOS
```
Going to manually merge this in. `macOS` is the right capitalization.